### PR TITLE
15836 update docs to reflect viewport updates

### PR DESCRIPTION
**Pull request by holdenhinkle**
ZenHub Ticket - https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/15836

Update the following sections to reflect recent code changes:

- Cypress Custom Commands > Viewport Presets: cy.viewportPreset(preset, orientation, options)
- Iterate Through Top VA.gov Viewports

Includes misc. edits.

### CODE REVIEW

File: platform/testing/end-to-end/cypress-best-practices-on-vsp.md [hunk 1]
```diff
@@ -335,105 +300,131 @@ https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/p
...
       `% traffic for the month of ${percentTrafficPeriod}: ${percentTraffic}%`
     );
 
-    cy.visit('/find-locations');
-    cy.injectAxe();
-    cy.viewport(width, height);
-    cy.checkSearch();
+    cy.visit('/page');
+    cy.viewportPreset(viewportPreset);
+    // code omitted
```
Comment (rjohnson2011) [inline platform/testing/end-to-end/cypress-best-practices-on-vsp.md:422]: Just checking in on this // code omitted comment.

**Review by rjohnson2011 — COMMENTED** [review]
Quick question about a commented out line.

File: platform/testing/end-to-end/cypress-best-practices-on-vsp.md (hunk 1 above)
Comment (holdenhinkle) [reply platform/testing/end-to-end/cypress-best-practices-on-vsp.md:422]: Good catch. For clarity, perhaps it should say `// test code omitted` or something like that?

**Review by 1Copenut — CHANGES_REQUESTED** [review]
Just one change requested. There's a couple of arguments for the `cy.axeCheck()` method that could be changed to a list to match the others. Screenshot attached below because I can't find the line number in the diff.

---
Screen Shot 2020-11-30 at 4 02 46 PM

**Comment by holdenhinkle** [comment]
@1Copenut Good catch. I just updated that.

**Review by 1Copenut — APPROVED** [review]
LGTM! Thanks @holdenhinkle

**Pull request merged**